opencti-platform/opencti-front/src/private/components/settings/SessionsList.jsx

```diff
@@ -163,14 +151,11 @@ class SessionsListComponent extends Component {
                   </ListItemIcon>
                   <ListItemText
                     primary={
-                      <div>
-                        <div className={classes.name}>{user.name}</div>
-                        <div className={classes.email}>{user.email}</div>
-                      </div>
+                      <div className={classes.name}>{user.name} (last 10 of {session.total} sessions)</div>
```

Comment:
TODO: need to change this part to display "last 10 of X" only when there are more then 10 sessions